### PR TITLE
Makes fresh h3agwas run on CloudOS with assoc test

### PR DESCRIPTION
**Pull request by Vlad-Dembrovskyi**
## Main:
The pipeline with latest changes from h3abionet made to run association part of GWAS workflow on CloudOS.
Successful CloudOS run with test data: https://cloudos.lifebit.ai/public/jobs/603fa68556ffbb0112ac9231
PR to fulfil [#14893](https://app.zenhub.com/workspaces/lifebit-platform-5aa6276b1224bb03d13f0e82/issues/lifebit-ai/lifebit-platform/14893).

## Details:
- original h3agwas is pulled with latest changes to current repository;
- main config file meanifest mainScript was parametrized to enable selecting different h3agwas workflows from command line;
- nextflow profiles re-defined to suite CloudOS, docker enabled in standard progile;
- local features that broke pipeline when running on CloudOS were resolved.

## To reproduce
To reproduce successful run locally:
```
nextflow run 'https://github.com/lifebit-ai/h3agwas' -r 9ce37c7ed90b95b349e94a113aed613f27df3faa -with-dag flowchart.png --data 's3://lifebit-featured-datasets/pipelines/gwas/testdata/sample-batch-site.phe' --pheno PHE --input_dir 's3://lifebit-featured-datasets/pipelines/gwas/testdata' --input_pat sampleA --covariates SEX,batch
```

### CODE REVIEW

File: assoc/main.nf [hunk 1]
```diff
@@ -53,10 +53,10 @@ params.each { parm ->
...
 
 
-params.queue      = 'batch'
-params.work_dir   = "$HOME/h3agwas"
-params.input_dir  = "${params.work_dir}/input"
-params.output_dir = "${params.work_dir}/output"
+//params.queue      = 'batch'
+//params.work_dir   = "$HOME/h3agwas"
+//params.input_dir  = "${params.work_dir}/input"
+//params.output_dir = "${params.work_dir}/output"
```
Comment (cgpu) [inline assoc/main.nf:59]: We can deleted alltogether if they are not needed.

```Suggestion

```

File: assoc/main.nf [hunk 2]
```diff
@@ -201,6 +202,7 @@ def fileColExists = { fname, pname, cname ->
 fileColExists(params.data,"${params.data} - covariates", params.covariates)
 fileColExists(params.data,"${params.data} - phenotypes", params.pheno)
 fileColExists(params.data,"${params.gxe} - gxe", params.gxe)
+*/
```
Comment (cgpu) [inline assoc/main.nf:205]: Long commented out section, we can also delete?

File: assoc/main.nf [hunk 3]
```diff
@@ -1481,8 +1484,9 @@ def getres(x) {
   res ="${proc.in.text}"
   return res.trim()
 }
+*/
 
-nextflowversion =getres("nextflow -v")
+nextflowversion = "Nextflow v${nextflow.version}"
```
Comment (cgpu) [inline assoc/main.nf:1489]: awesome fetch of version Vlad! 🚀

**Review by cgpu — APPROVED** [review]
Looks good to me!

I added comments for deleting the commented out sections.
Though they may put us out of sync and create conflicts, it's better to delete them if we maintain it.

**Comment by Vlad-Dembrovskyi** [comment]
Deleted commented-out parts as suggested.

**Pull request merged**